src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
+@import './utils/variable';
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), file extensions in import statements should be omitted. Use `@import './utils/variable';` instead of `@import './utils/variable.scss';`.